test/unit/rules/no-capital-arguments-test.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const generateRuleTests = require('../../helpers/rule-test-harness');
+
+const { ERROR_MESSAGE } = require('../../../lib/rules/no-capital-arguments');
+
+generateRuleTests({
+  name: 'no-capital-arguments',
+
+  config: true,
+
+  good: ['<Foo @name="bar" />'],
+
+  bad: [
+    {
+      template: '<Foo @Name="bar" />',
+      fixedTemplate: '<Foo @name="bar" />',
```

Comment:
fixers should be safe, because capital arguments is not working :)

Reply:
No, the fixer is not guaranteed to be safe. The fixer as implemented assumes that the only possible way to use a passed in named argument is via named arguments in the template of the component you are calling (which is not true). For example, `Foo` component in this case could easily be using `this.args.Name` in its JavaScript portion.

Reply:
@rwjblue I have added linting for `paths` as well, could we use fixer for paths?